python-frontend/src/main/java/org/sonar/python/types/v2/UnknownType.java

```diff
@@ -36,5 +40,12 @@ final class UnknownTypeImpl implements UnknownType {
   }
 
   record UnresolvedImportType(String importPath) implements UnknownType {
+    @Override
+    public Optional<PythonType> resolveMember(String memberName) {
+      var memberFqn = Stream.of(importPath, memberName)
+        .filter(Predicate.not(String::isEmpty))
+        .collect(Collectors.joining("."));
+      return Optional.of(new UnresolvedImportType(memberFqn));
+    }
```

Comment:
I think we might want to extend this idea in the future, by allowing the creation of a "fake" type hierachy.